spinn_front_end_common/interface/spinnaker.cfg

```diff
@@ -106,6 +106,8 @@ spalloc_machine = None
 
 virtual_board = False
 requires_wrap_arounds = None
+NCoresPerChip = 18
```

Comment:
Would it be better to make this 16 by default since this is what it was in the current code?

Reply:
but i takes out one core to monitor in spinnMachine. so yourd really argue 17, but i dont know. ive put it as 16 anyhow.